examples/kubernetes/connectivity-check/connectivity-check.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: probe
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        name: probe
+    spec:
+      containers:
+      - name: probe-container
+        image: docker.io/cilium/json-mock
```

Comment:
Please add a image that that is not `latest`

Also, in docker hub create an automatic build so we can have the option to modify the code, push it to github and release a new version if needed.

Reply:
Usually I would agree but this image does not run custom code of ours. The Dockerfile is:
```
FROM node:latest
RUN npm install -g json-server curl

ADD run.sh default.json /
ENTRYPOINT ["bash", "/run.sh"]
CMD []
```

I don't expect this container image to ever be unstable or in development phase. I agree with the automatic build though.

Reply:
Setup an automated build

Reply:
@aanm Yes, that makes sense

Reply:
Set to 10.5.0-slim